pipeline/outputs/stackdriver.md

```diff
@@ -78,6 +79,75 @@ This implies that if there is no local_resource_id in the log entry then the tag
     tag_prefix  custom_tag.
 ```
 
+## Resource Labels
+
+Currently, there are four ways which fluent-bit uses to assign fields into the resource/labels section.
+
+1. Resource Labels API
+2. Monitored Resource API
+3. Local Resource Id
+4. Credentials / Config Parameters
+
+If `resource_labels` is correctly configured, then fluent-bit will attempt to populate all resource/labels using the entries specified. Otherwise, fluent-bit will attempt to use the monitored resource API. Similarly, if the monitored resource API cannot be used, then fluent-bit will attempt to populate resource/labels using configuration parameters and/or credentials specific to the resource type. As mentioned in the [Configuration File](#configuration-file) section, fluent-bit will attempt to use or construct a local resource ID for a K8s resource type which does not use the resource labels or monitored resource API.
+
+Note that the `project_id` resource label will always be set from the service credentials or fetched from the metadata server and cannot be overridden.
+
+### Using the resource_labels parameter
+
+The `resource_labels` configuration parameter offers an alternative API for assigning the resource labels. To use, input a list of comma separated strings specifying resource labels plaintext assignments (new=value), mappings from an original field in the log entry to a destination field (destination=$original) and/or environment variable assignments (new=${var}).
+
+For instance, consider the following log entry:
+```
+{
+  "keyA": "valA",
+  "toplevel": {
+    "keyB": "valB"
+  }
+}
+```
+
+Combined with the following Stackdriver configuration:
+```
+[OUTPUT]
+    Name stackdriver
```

Comment:
The `resource` parameter defaults to `global`, which is what is used in the example (see L121). Do you think it would be beneficial to mention the `resource` parameter?

Reply:
An example with specified resource could also be helpful.

Reply:
I've updated the last example to contain a sample configuration for `k8s_container`.